### PR TITLE
Document changes from the last 4 PRs

### PR DESCRIPTION
**Pull request by byshy**
- Add documentation for the fix command
- Add documentation for the no-blank-line-before-single-return rule
- Add documentation for the prefer-first-or-null rule
- Add documentation for the prefer-single-quotes rule

### CODE REVIEW

File: docs/rules/dart/prefer-first-or-null.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# prefer-first-or-null
+added in: 1.6.1 <span style={{color: 'green'}}>style</span>
```
Comment (santitigaga) [inline docs/rules/dart/prefer-first-or-null.mdx]: the new release will be 1.2.0 please

File: docs/rules/dart/prefer-single-quotes.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# prefer-single-quotes
+added in: 1.6.1 <span style={{color: 'green'}}>style</span>
```
Comment (santitigaga) [inline docs/rules/dart/prefer-single-quotes.mdx]: the new release will be 1.2.0 please

File: docs/rules/dart/no-blank-line-before-single-return.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# no-blank-line-before-single-return
+added in: 1.6.1 <span style={{color: 'green'}}>style</span>
```
Comment (santitigaga) [inline docs/rules/dart/no-blank-line-before-single-return.mdx]: the new release will be 1.2.0 please

File: docs/cli/fix.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Analyze
+
+Reports lint rules violations.
+
```
Comment (santitigaga) [inline docs/cli/fix.md:6]: It would be helpful to add the version in which this task is added as well

**Review by santitigaga — CHANGES_REQUESTED** [review]

**Comment by byshy** [comment]
Will fix it shortly, just a quick note, after inspecting other files I saw the version 1.6.0, this is why I opted to using 1.6.1, do you want to check the other ones in another PR?

**Comment by santitigaga** [comment]
@byshy  you are right, all rules are available since the first release, That version 1.6.0 is a bug that should have been left from the fork. At least adjust the contribution, I will adjust the others in the next few days.

**Comment by byshy** [comment]
The version in the PR is fixed and the PR is ready

**Review by santitigaga — APPROVED** [review]
Thank you

**Pull request merged**